modules/cloudevent-recorder/recorder.tf

```diff
@@ -95,6 +95,14 @@ module "triggers" {
   notification_channels = var.notification_channels
 }
 
+locals {
+  alerts = tomap({
+    for type, schema in var.types :
+    "BQ DTS ${var.name}-${type}" => google_monitoring_alert_policy.bq_dts[type].id
```

Comment:
Is the idea that this is a temporary step to allow folks to `terraform import` the resource and get back into the normal state?

It'd be nice if we could get the alerts even on existing datasets.

Reply:
yeah, I've just made a change, sorry muscle memory amended the previous commit and pushed so it's not obvious my change but if you look at the 
```
locals {
  alerts = tomap({....
```
you can see the change, is that what you're thinking?

Reply:
I'd love for the goal-state to be what we have today, and not need to keep this bridge logic around ~forever.

Can we import the old data into a new dataset?  Can we switch consumers to use a `UNION` query for a bit?

Reply:
There are other defaults like dataset and table expiration that he current tables are created with by default.  The usecase we have for image scans means we'd want to not expire that data.  So being able to provide as existing dataset and tables means folks can set these up with whatever config they want and still benefit from these modules.

We could change downstream queries to handle a `UNION`, or transfer data but that increases the bar to migrate to common infra, which has taken some time already.  If it's an issue then yeah we can do it and will look into that next week, just a lot is depending on a migration asap really.